azurelinuxagent/common/protocol/goal_state.py

```diff
@@ -36,20 +37,24 @@
 TRANSPORT_PRV_FILE_NAME = "TransportPrivate.pem"
 
 
+class GoalStateRetrievalMode:
```

Comment:
Let's document what those values mean

Reply:
Or: this "enum" seems to be map 1:1 with WireClient:_UpdateType... can you merge them into a single enum?

Reply:
merge with UpdateType